src/approaches/oracle_approach.py

```diff
@@ -851,9 +853,11 @@ def movedoortodial_sampler(state: State, rng: np.random.Generator,
     door = Variable("?door", door_type)
     region = Variable("?region", region_type)
     parameters = [robot, dial, door, region]
-    option_vars = [robot]
+    option_vars = [robot, door, region]
```

Comment:
if you still have trouble with learning, one reason may be that if we add these option vars, they need to be used somehow by the option itself. otherwise, in the data, you could see strange things like `Move(robot1, door1, region1)` resulting in a change to `door5` and `region6`. probably the easiest way to prevent this would be to change the `initiable` for the options so that they use the option vars somehow. for example, you could check that `door` and `region` are in the playroom, in this case.